include/libtorrent/torrent.hpp

```diff
@@ -95,6 +95,8 @@ namespace libtorrent
 	class bt_peer_connection;
 	struct listen_socket_t;
 
+	typedef std::chrono::duration<std::int32_t> seconds32;
```

Comment:
if you'll make another change, please make this: ``using seconds32 = std::chrono...``